recipes/OpenPatchMiner/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "OpenPatchMiner" %}
+{% set version = "0.1.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/OpenPatchMiner-{{ version }}.tar.gz
+  sha256: 11b9089a0f55ec50b61fa002acf527299191af344aebb44053cbb021c8521355
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - keyring
+    - matplotlib-base
+    - numpy
+    - openslide-python
+    - pandas
+    - python >=3.6
+    - scikit-image
+    - tqdm
+    - twine
+    - wheel
+
+test:
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/CBICA/OPM
+  summary: A patch miner for large histopathology images
+  license: LicenseRef-CBICA
```

Should be this now, right?
```suggestion
  license: BSD-3-Clause
```